test/runnable/testgc3.d

```diff
@@ -1,5 +1,11 @@
 // PERMUTE_ARGS:
 // REQUIRED_ARGS:
+/*
```

Comment:
Would be nice to merge  this  with the previous comments (`//`). Here and everywhere else.

Reply:
Can do, but that would be a very tedious endeavor ...